### PR TITLE
Fixed MSVC warnings

### PR DESCRIPTION
**Pull request by arummler**
There are still about 28 left. More commits to come.

### CODE REVIEW

**Comment by simonspa** [comment]
okay, waiting with merge then. Thanks for the work!

**Comment by arummler** [comment]
@simonspa You are welcome. Now down to the 11 in the converter plugins (the bit shift operations, arghh) which are mentioned in the issues - please pull. I will add the changes as well to 1.5 and open a new PR.

File: main/exe/src/TestRunControl.cxx [hunk 1]
```diff
@@ -57,8 +57,8 @@ void * worker_automatic_run(void * arg) {
 
 
         rc->StartRun("");
-        std::cout << "sleep for " << duration << std::endl;
-        eudaq::mSleep(duration);
+        std::cout << "sleep for " << (unsigned int) duration << std::endl;
+        eudaq::mSleep((unsigned int)duration);
```
Comment (simonspa) [inline main/exe/src/TestRunControl.cxx]: don't we want proper casting via `static_cast<>()`?

**Comment by arummler** [comment]
You are right - I was using the sloppy old C style casting. But I think I did it at other places as well, let me check before merging.

**Comment by arummler** [comment]
Done

**Comment by arummler** [comment]
Appveyor build fails with code 1 --> When I connect with rdesktop to the virtual machine it works. There are some older thread in the help board which look similar, probably I will open also one.

**Comment by arummler** [comment]
Need to check first why it is failing.

**Pull request merged**